src/extension/manifest.json

```diff
@@ -56,7 +56,8 @@
           "tab.js",
           "url-cache.js",
           "view/json/json.html",
-          "view-doc-source/js/content.js"
+          "view-doc-source/js/content.js",
+          "icon-32x32.png"
```

Comment:
Shouldn't this be 
```suggestion
          "icons/default/icon-32x32.png"
```
PS: Let's keep this list in alphabetical order

Reply:
Rollup pulls in out of `/icons/default` and puts it into the root. The reference in the build becomes  

```html
<img alt="adobe logo" src="chrome-extension://iijidanmjddcjacejcjofhcjmgkkiijo/icon-32x32.png">
```

Didn't [see a way](https://modern-web.dev/docs/building/rollup-plugin-import-meta-assets/) to control where it's put or where it's referenced from. 

Thinking now though this should actually be an SVG so it looks crisp on HDPI screens. Do you have the svg for this black version of the icon @rofe?